CMakeLists.txt

```diff
@@ -17,19 +20,35 @@ include(AsteroidCMakeSettings)
 set(ASTEROID_MODULES_INSTALL_DIR ${CMAKE_INSTALL_DATADIR}/asteroidapp/cmake)
 
 find_package(Qt5 ${QT_MIN_VERSION} COMPONENTS DBus Qml Quick Svg REQUIRED)
-find_package(Mlite5 MODULE REQUIRED)
-find_package(Mapplauncherd_qt5 MODULE REQUIRED)
+if (WITH_DEMO)
+    find_package(Mlite5 MODULE REQUIRED)
+    find_package(Mapplauncherd_qt5 MODULE REQUIRED)
+endif()
 
 ecm_find_qmlmodule(QtQuick.VirtualKeyboard 2.1)
+if (NOT WITH_CMAKE_MODULES)
```

Comment:
Could you move this into an else clause of the if WITH_CMAKE_MODULES ? It would be a bit easier to read that we go through one or the other

Reply:
Done.